src/material-examples/form-field-prefix-suffix/form-field-prefix-suffix-example.html

```diff
@@ -1,7 +1,9 @@
 <div class="example-container">
   <mat-form-field>
     <input matInput placeholder="Enter your password" [type]="hide ? 'password' : 'text'">
-    <mat-icon matSuffix (click)="hide = !hide">{{hide ? 'visibility_off' : 'visibility'}}</mat-icon>
+    <button mat-icon-button matSuffix (click)="hide = !hide" aria-label="Show or hide password icon">
```

Comment:
The `aria-label` show be different based on whether the button is going to show or hide the value:

```
[attr.aria-label]="hide ? 'Hide password' : 'Show password'"
````

Reply:
Changed as per your suggestion. Can you please check now